### ISSUE TITLE
FEATURE - User Flow Diagram

### ISSUE THREAD

**Issue by rajhraval** [mikaelacaron/Basic-Car-Maintenance#87]
## Describe the feature you want the app to have that it currently doesn't
A feature list and user flow diagram to understand the app working.

## Describe alternatives you've considered
It can be a simple flow diagram,, which describes what's the app all about for newcomers.

**Comment by mikaelacaron** [comment]
this would be great to add to the DocC Documentation!

**Comment by ItsDastagir333** [comment]
can I work on it??

**Comment by clydehenry3** [comment]
I would love to take this up.

**Comment by mikaelacaron** [comment]
@clydehenry3 can you first do  #40 ? before I assign another issue to you

**Comment by Artal44** [comment]
Is this task still available ?

**Comment by mikaelacaron** [comment]
@Artal44  Overall yes, but this may be hard to do at the moment because of the changes I'm making before I ship the app. Once #260 is complete, then you could do this one

**Comment by Artal44** [comment]
Do you know how much time it is going to take ? Otherwise i can maybe already start with a first version ?

**Comment by mikaelacaron** [comment]
The goal is the end of the month, you can start on this task, but it's going to change, whenever I finish #260, and I would prefer you'd wait, so I don't have to go back and point out all the changes

**Comment by Artal44** [comment]
Okay i will wait for it no problem

**Comment by Artal44** [comment]
Hi, Do you have any news regarding the end of #260 ?

**Comment by mikaelacaron** [comment]
Hey I had something come up, and am going to move my deadline to end of the year! I appreciate you wanting to help, but this issue can't be completed at this time

**Comment by horacio3m** [comment]
Is this task open?

**Comment by mikaelacaron** [comment]
@horaciomuller 

What ideas do you have to complete this task?

Also all tasks will start on Oct 1 for Hacktoberfest

**Comment by mikaelacaron** [comment]
Going to close this for now, cause I'm not really sure where the direction of it is
